### PR TITLE
New param in the firewall to avoid  the creation of the default rules

### PR DESCRIPTION
**Pull request by alejandrojnm**
Added the option to not create the default rules when you create a firewall

Signed-off-by: Alejandro J. Nuñez Madrazo <alejandrojnm@gmail.com>

### CODE REVIEW

**Comment by andyjeffries** [comment]
I don't understand, the new field is a string? It's not documented anywhere (is it?) what values SHOULD go in there in a normal case nor what value to put in there to "not create the default rules when you create a firewall".

File: firewall.go [hunk 1]
```diff
@@ -54,9 +54,10 @@ type FirewallRuleConfig struct {
 
 // FirewallConfig is how you specify the details when creating a new firewall
 type FirewallConfig struct {
-	Name      string `json:"name"`
-	Region    string `json:"region"`
-	NetworkID string `json:"network_id"`
+	Name        string `json:"name"`
+	Region      string `json:"region"`
+	NetworkID   string `json:"network_id"`
+	CreateRules bool   `json:"create_rules,omitempty"`
```
Comment (RealHarshThakur) [inline firewall.go:60]: Shouldn't be a pointer here too?

Comment (RealHarshThakur) [reply firewall.go:60]: For reference, a comment would be nice too. 
If `nil` is sent, it implies rules will be created. 
Essentially, unless user explicitly sends false, we'll create rules for them

Comment (RealHarshThakur) [reply firewall.go:60]: By default the variable will be set to false, won't it?

Comment (alejandrojnm) [reply firewall.go:61]: Yes true

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**